tests/publiccloud/slem_basic.pm

```diff
@@ -22,15 +22,20 @@ sub run {
     my $provider = $self->provider_factory();
     $provider->{username} = 'suse';
     my $instance = $self->{my_instance} = $provider->create_instance(check_guestregister => 0);
-    my $test_package = 'strace';
+    my $test_package = 'jq';
     registercloudguest($instance);
     $instance->run_ssh_command(cmd => 'zypper lr -d', timeout => 600);
     $instance->run_ssh_command(cmd => 'systemctl is-enabled issue-generator');
     $instance->run_ssh_command(cmd => 'systemctl is-enabled transactional-update.timer');
     $instance->run_ssh_command(cmd => 'systemctl is-enabled issue-add-ssh-keys');
-    $instance->run_ssh_command(cmd => 'sudo transactional-update -n pkg install ' . $test_package, timeout => 600);
-    $instance->softreboot();
-    $instance->run_ssh_command(cmd => 'rpm -q ' . $test_package);
+
+    my $ret = $instance->run_ssh_command(cmd => 'rpm -q ' . $test_package, rc_only => 1);
+    if ($ret) {
```

Comment:
I wonder if we should fail if the package exist, so we make sure we test always this block.

Reply:
I mean, if it fails, it means we need to search for another package and change it here :)

Reply:
hm...sure, I can change that.

Reply:
done